### PR TITLE
Replace "DanubeTech" with "verifiable-credentials-java" test report.

### PR DESCRIPTION
**Pull request by peacekeeper**
Lists [verifiable-credentials-java](https://github.com/danubetech/verifiable-credentials-java) instead of "DanubeTech".

### CODE REVIEW

File: implementations/index.html [hunk 1]
```diff
@@ -1181,7 +1181,7 @@ <h2>Credential Status (optional)</h2>
 <table class="simple">
   <thead>
     <th width="80%">Test</th>
-<th>BrightLink</th><th>Credly</th><th>DanubeTech</th><th>Evernym</th><th>Factom-Harmony-Integrate</th><th>Guillaume-Daumas</th><th>Sovrin-Ken_Ebert</th><th>Toulouse-Kent</th><th>uPort-kotlin</th><th>uPort</th><th>vc.js</th>
+<th>BrightLink</th><th>Credly</th><th>Evernym</th><th>Factom-Harmony-Integrate</th><th>Guillaume-Daumas</th><th>Sovrin-Ken_Ebert</th><th>Toulouse-Kent</th><th>uPort-kotlin</th><th>uPort</th><th>vc.js</th><th>verifiable-credentials-java</th>
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 2]
```diff
@@ -170,7 +170,7 @@ <h2>Basic Documents</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 3]
```diff
@@ -1274,7 +1274,7 @@ <h2>Advanced Documents</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 4]
```diff
@@ -1610,7 +1610,7 @@ <h2>Linked Data Proofs (optional)</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 5]
```diff
@@ -1703,7 +1703,7 @@ <h2>Credential Schema (optional)</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 6]
```diff
@@ -1823,7 +1823,7 @@ <h2>Refresh Service (optional)</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 7]
```diff
@@ -1943,7 +1943,7 @@ <h2>Terms of Use (optional)</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 8]
```diff
@@ -2009,7 +2009,7 @@ <h2>Evidence (optional)</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 9]
```diff
@@ -2075,7 +2075,7 @@ <h2>JWT (optional)</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

File: implementations/index.html [hunk 10]
```diff
@@ -2897,7 +2897,7 @@ <h2>Zero-Knowledge Proofs (optional)</h2>
...
```
Comment (TallTed) [inline implementations/index.html]:
```suggestion
<th>BrightLink</th>
<th>Credly</th>
<th>Evernym</th>
<th>Factom-Harmony-Integrate</th>
<th>Guillaume-Daumas</th>
<th>Sovrin-Ken_Ebert</th>
<th>Toulouse-Kent</th>
<th>uPort-kotlin</th>
<th>uPort</th>
<th>vc.js</th>
<th>verifiable-credentials-java</th>
```

**Comment by msporny** [comment]
@TallTed - rejecting your change set -- you're trying to modify an auto-generated file.

**Comment by msporny** [comment]
@peacekeeper -- merge conflicts.

**Comment by peacekeeper** [comment]
@msporny merge conflicts resolved.

**Comment by TallTed** [comment]
@peacekeeper - you might need to take another whack at merge conflicts. (Github shows me [43 conflict blocks](https://github.com/w3c/vc-test-suite/pull/116/conflicts).)

**Comment by TallTed** [comment]
@msporny -
> @TallTed - rejecting your change set -- you're trying to modify an auto-generated file.

Bah. Fair enough. I wonder if there's a way to blatantly flag auto-generation within the output files?

**Comment by peacekeeper** [comment]
Merge conflicts resolved again :)

**Comment by msporny** [comment]
>  I wonder if there's a way to blatantly flag auto-generation within the output files?

I know of no way to do this AND get that warning in front of people that need to see it -- the PR wouldn't contain the warning since the warning doesn't change. Happy to implement something if I knew what to implement.

**Comment by TallTed** [comment]
@msporny - Maybe in the file or directory names?  "implementations-autogen/index.html" or "implementations/index-autogen.html" or "autogen/implementations-report/index.html"... Just thinking out loud toward the future...

**Comment by msporny** [comment]
> Just thinking out loud toward the future...

Ideally, we overhaul the entire test suite so it's run on a nightly basis against the latest implementations... and all auto-generated content is placed outside of the repository (with a link to a history of all the interop tests and the latest interop tests on the landing page for the test suite).

**Pull request merged**